cornice/schemas.py

```diff
@@ -175,3 +175,78 @@ def _validate_fields(location, data):
         for param in set(params) - set([attr.name for attr in attrs]):
             request.errors.add('body' if param in body else 'querystring',
                                param, msg % param)
+
+
+def validate_matchdict(matchdict_types):
+    """Validates types of request.matchdict through matchdict_types
+    dictionary.
+
+    matchdict_types dictionary should have it's values defined as
+    callable type. Here's an example::
+
+        @view(
+            ...
+            validators=( validate_matchdict( {'id': int} ), ),
+            ...
+        )
+    """
+    def wrapper(request):
+        md = request.matchdict
```

Comment:
You mean the module for extension or method? I don't think this should be submitted_data, because this method relies upon matchdict which does not have anything to do with submitted_data (i.e. POST/PUT form data). Or are you referring to validate_wtforms_schema?